registry.rst

```diff
@@ -0,0 +1,84 @@
+.. _registry:
+
+####################
+OCI Image Registries
+####################
+
+It is common for users of {Singularity} to use `OCI
+<https://opencontainers.org/>`__ registries as sources for their container
+images. Some registries require credentials to access certain images or even the
+registry itself. Previously, the only method in {Singularity} to supply
```

Comment:
This and the `Starting with Singularity 4.0` bit sort of infer that the `registry login` is entirely moved, rather than new.

At this point I think we can drop the `previously` / `starting with` stuff, and just list the ways that there are to provide credentials?